controllers/component_build_controller_pac.go

```diff
@@ -148,6 +156,253 @@ func (r *ComponentBuildReconciler) ProvisionPaCForComponent(ctx context.Context,
 	return mrUrl, nil
 }
 
+func getHttpClient() *http.Client {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: gp.IsInsecureSSL()},
```

Comment:
We might need to mark it as an exception with a comment explaining why we do it.